### DISCUSSION TITLE
Split menu dropdowns in Bootstrap

### DISCUSSION THREAD

**Discussion by SoftCircuits** [twbs/bootstrap#34305] [Q&A]
So, I was very disappointed to learn that submenus have been removed from Bootstrap. I have an increasingly complicated website, and submenus would be a clean way to do what I need.

Another option would be if I could implement split dropdowns in the navbar. That is, have menus where, if you click the main part, it runs the most common related command, and if you click the down arrow part, it would dropdown with additional commands. It would work the same as [split button dropdowns](https://getbootstrap.com/docs/4.0/components/dropdowns/#split-button-dropdowns).

Does anyone know if there's any support for split menu dropdowns?

**Comment by mandrasch** [comment]
Hey, 

wondered about this too. I used a split button here https://mandrasch.github.io/11ty-plain-bootstrap5/ with the help of `btn.group`.

Styling is a bit off and I'll need to check if it is fully accessible. 

```
<li class="nav-item dropdown">
   <div class="btn-group">
      <a class="nav-link" href="/11ty-plain-bootstrap5/pages/examples/" data-bs-toggle="Examples" aria-expanded="false" id="dropdown-804376">
      Examples
      </a><a class="nav-link dropdown-toggle dropdown-toggle-split" data-bs-toggle="dropdown" href="#" aria-expanded="false">
      <span class="visually-hidden">Toggle Dropdown</span>
      </a>
      <ul class="dropdown-menu" aria-labelledby="dropdown-804376">
         <li>    
            <a class="dropdown-item" href=" /11ty-plain-bootstrap5/pages/examples/cover/">
            Cover
            </a>
         </li>
      </ul>
   </div>
   <!-- eo btn-group for dropdowns -->
</li>
```
